platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/store/SystemConfigurationMigratorTest.java

```diff
@@ -0,0 +1,336 @@
+
+    private static final String TRUSTSTORE_SYSTEM_PROP = "javax.net.ssl.trustStore";
+
+    private static final String WS_SECURITY_DIR_REL_PATH = "etc/ws-security";
+
+    private static final String PDP_POLICIES_DIR_REL_PATH = "etc/pdp";
+
+    private static final String SYSTEM_PROPERTIES_REL_PATH = "etc/system.properties";
+
+    private static final String USERS_PROPERTIES_REL_PATH = "etc/users.properties";
+
+    private static final String VALID_CRL_REL_PATH_1 = "etc/certs/demoCA/crl/crl.pem";
+
+    @Before
+    public void setup() {
+        mockStatic(System.class);
+        mockStatic(FileUtils.class);
+    }
+
+    @Test
+    public void testExportValidRelativePaths() throws Exception {
+        // Setup
+        when(System.getProperty(KEYSTORE_SYSTEM_PROP)).thenReturn(KEYSTORE_REL_PATH);
+        when(System.getProperty(TRUSTSTORE_SYSTEM_PROP)).thenReturn(TRUSTSTORE_REL_PATH);
+        Stream<String> stream = getStream(ENC_PROP_FILE_REL_CRL_PATH);
+        doNothing().when(FileUtils.class);
+        SystemConfigurationMigrator securityConfigurationMigrator = new SystemConfigurationMigrator(
+                DDF_HOME) {
+            @Override
+            Stream<String> getStream(Path path) throws IOException {
+                return stream;
+            }
+        };
+
+        // Perform Test
+        securityConfigurationMigrator.export(EXPORT_DIRECTORY);
+
+        // Verify
+        ArgumentCaptor<File> sourceDirCaptor = ArgumentCaptor.forClass(File.class);
+        ArgumentCaptor<File> destinationDirCaptor = ArgumentCaptor.forClass(File.class);
+        verifyStatic(times(2));
+        FileUtils.copyDirectory(sourceDirCaptor.capture(), destinationDirCaptor.capture());
+        List<File> sourceDirs = sourceDirCaptor.getAllValues();
+        List<File> destinationDirs = destinationDirCaptor.getAllValues();
+        assertThat(sourceDirs.get(0).getAbsolutePath(), is(DDF_HOME + File.separator
+                + WS_SECURITY_DIR_REL_PATH));
+        assertThat(destinationDirs.get(0).getAbsolutePath(), is(EXPORT_DIRECTORY + File.separator
+                + WS_SECURITY_DIR_REL_PATH));
+        assertThat(sourceDirs.get(1).getAbsolutePath(), is(DDF_HOME + File.separator
+                + PDP_POLICIES_DIR_REL_PATH));
+        assertThat(destinationDirs.get(1).getAbsolutePath(), is(EXPORT_DIRECTORY + File.separator
+                + PDP_POLICIES_DIR_REL_PATH));
+
+        ArgumentCaptor<File> sourceFileCaptor = ArgumentCaptor.forClass(File.class);
+        ArgumentCaptor<File> destinationFileCaptor = ArgumentCaptor.forClass(File.class);
+        List<File> sourceFiles = sourceFileCaptor.getAllValues();
+        List<File> destinationFiles = destinationFileCaptor.getAllValues();
+        verifyStatic(times(5));
+        FileUtils.copyFile(sourceFileCaptor.capture(), destinationFileCaptor.capture());
+        assertThat(sourceFiles.get(0).getAbsolutePath(), is(DDF_HOME + File.separator
```

Comment:
Same comment as above. And, as a bonus, if you can get that to work you won't need the `verifyStatic(times(5))` anymore :-)